res/old/1/res_cia_12_50.csv

```diff
@@ -0,0 +1,4 @@
+870192.3446021639
```

Comment:
Do we want to version the results?

Reply:
Maybe, to have them fixed and available for the next iteration of the paper.